frontend/src/metabase/public/components/EmbedModal/SelectEmbedTypePane/InteractiveEmbeddingCTA/InteractiveEmbeddingCTA.tsx

```diff
@@ -12,6 +13,9 @@ import {
 
 const useCTAText = () => {
   const isPaidPlan = useSelector(getIsPaidPlan);
+  const userPlan = useSelector(state =>
```

Comment:
nit: it's not really the plan of the _user_, more of the instance.
WDYT about just "plan"?

Reply:
Fair enough, I saw someone else use `plan` for the variable name but I have a (most likely unfounded) annoyance towards single word variable names 🙃 . I'll change it